FinniversKit/Sources/Fullscreen/FrontPage/FrontPageView.swift

```diff
@@ -269,6 +298,18 @@ public final class FrontPageView: UIView, BasicFrontPageView {
         
     }
     
+    private func addFrontPageShelf() {
```

Comment:
I noticed we call this function every time the view model changes (and thus remove the full view hierarchy and add it again). We should reload the shelf collection view instead, less expensive 😄